demo.scenario

```diff
@@ -78,7 +78,7 @@ $ docker images | grep redis
 debian-redis  latest    8366943c77e8     3 days ago       136MB
 alpine-redis  latest    aae644cd3417     3 days ago       6.99MB
 nix-redis     latest    30486183a209     50 years ago     45.4MB
-$ # Size of a Nix based docker image is somewhere between a Debian and Alpine
+$ # The size of a Nix based docker image is somewhere between a Debian and an Alpine image
```

Comment:
We need to make this git into 77 character in a line. I found 77 chars to produce the most readable video, while still having some space to write show things. Can you combine this with next line and make sure you don't go over 77 chars? even if we need to add additional line